mujoco_ros/launch/mushr_feedback_example.launch

```diff
@@ -0,0 +1,10 @@
+<launch>
+    <param name="model_path" type="string" value="$(find mj_models)/models/mushr/mushr.xml"/>
+    <group ns="mushr">
+        <node pkg="mujoco_ros" name="mushr_feedback" type="mushr_feedback" output="screen" >
+            <param name="visualize" value="true" />
+        </node>
+        <!-- <node pkg="mujoco_ros" name="keyboard_service" type="keyboard_service.py" output="screen" launch-prefix="xterm -e"/> -->
```

Comment:
Why is this removed?

Reply:
In some setups, is easier to launch this in a separate terminal (and the launch-prefix for a separate terminal not always works)